pkg/handler/v2/group-project-binding/groupprojectbinding.go

```diff
@@ -59,3 +59,66 @@ func GetGroupProjectBindingEndpoint(
 		)
 	}
 }
+
+func CreateGroupProjectBindingEndpoint(
+	userInfoGetter provider.UserInfoGetter,
+	projectProvider provider.ProjectProvider,
+	privilegedProjectProvider provider.PrivilegedProjectProvider,
+	bindingProvider provider.GroupProjectBindingProvider,
+) endpoint.Endpoint {
+	return func(ctx context.Context, req interface{}) (interface{}, error) {
+		if err := createGroupProjectBinding(
+			ctx,
+			req,
+			userInfoGetter,
+			projectProvider,
+			privilegedProjectProvider,
+			bindingProvider,
+		); err != nil {
+			return nil, err
+		}
+		return nil, nil
+	}
```

Comment:
NIT:

maybe simpler would be:
```suggestion
		return nil, createGroupProjectBinding(
			ctx,
			req,
			userInfoGetter,
			projectProvider,
			privilegedProjectProvider,
			bindingProvider)
	}
```